documentation/src/docs/security/securing-distributed-mode-cluster.md

```diff
@@ -310,106 +103,134 @@ Controller services can be configured in three different ways:
+ | `autoScale.controller.connect.security.tls.validateHostName.enable` | `true` |
+ | `pravega.client.auth.loadDynamic` | `false` |
+ | `pravega.client.auth.method` | `Basic` |
+ | `pravega.client.auth.token` | Base64-encoded value of 'username:password' string | 
+
+[1]: It is assumed here that Zookeeper TLS is disabled. You may enable it and specify the corresponding client-side TLS 
+configuration properties via the `pravegaservice.zk.*` properties.
 
 [2]: The secret value you use here must match the same value used for other Controller and Segment Store services.
 
 ### Configuring TLS and Credentials on Client Side
 
-After enabling and configuring TLS and auth on the server-side services, its time to update the clients,
-so that the they can establish TLS connections with the servers and are allowed access.
-
-For TLS, establish trust for the servers' certificates on the client side using one of the following ways:
+After enabling and configuring TLS and auth on the server-side services, it's time for the clients' setup.
 
-  1. Supply the client library with the certificate of the trusted CA that has signed the servers' certificates.
+Clients can be made to trust the server's certificates signed by custom CA's using one of the following ways:
 
-     ```
+  1. Configure the client application to use the signing CA's certificate as the truststore. Alternatively, use the 
+     servers' certificate as the truststore. 
+     
+     ```java
      ClientConfig clientConfig = ClientConfig.builder()
-                .controllerURI("tls://<DNS-NAME-OR-IP>:9090")
-                .trustStore("/etc/secrets/ca-cert")
+                .controllerURI("tls://<dns-name-or-ip>:9090")
+                .trustStore("/etc/secrets/ca-cert.crt")
                 ...
                 .build();
      ```
-
-  2. Install the CA's certificate in the Java system key store.
+  2. Install the CA's certificate in the Java system key store. 
   3. Create a custom truststore with the CA's certificate and supply it to the Pravega client application,
-   via JVM system properties `javax.net.ssl.trustStore` and `javax.net.ssl.trustStorePassword`.
+     via standard JVM system properties `javax.net.ssl.trustStore` and `javax.net.ssl.trustStorePassword`.
 
 For auth, client-side configuration depends on the `AuthHandler` implementation used. If your server is configured to
-use the default `PasswordAuthHandler`, you may supply the credentials as shown below.
+use the built-in Password Auth Handler that supports "Basic" authentication, you may supply the credentials as shown below.
 
-  ```
+  ```java
   ClientConfig clientConfig = ClientConfig.builder()
-                .controllerURI("tls://<DNS-NAME-OR-IP>:9090")
-                .trustStore("/etc/secrets/ca-cert")
+                .controllerURI("tls://<dns-name-or-ip>:9090")
+                .trustStore("/etc/secrets/ca-cert.crt")
                 .credentials(new DefaultCredentials("changeit", "marketinganaylticsapp"))
```

Comment:
make it more obvious that `marketinganaylticsapp` is the username... I had to look up that this is `DefaultCredentials(password, username)`.

Reply:
Sure, I have changed it to "<password>" and "<username>" now, which should make the order of the two arguments more explicit. Thanks for your meticulous review!